mapping-for-navigation/_posts/0000-01-10-circular-junctions.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Modeling Circular Junctions - Roundabouts
+---
+
+A **roundabout** is a type of circular junction where multiple roads meet at a junction and the one-way traffic flow is directed around the island at the center.
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/9/9d/Notviken.jpg/1200px-Notviken.jpg)
+<sup>Aerial view of [Notviken in Luleå](https://sv.wikipedia.org/wiki/Notviken). Wikipedia</sup>
+
+> Studies by the Federal Highway Administration have found that roundabouts can increase traffic capacity by 30 percent to 50 percent compared to traditional intersections.
+
+ <sup> \*&nbsp;<https://www.wsdot.wa.gov/Safety/roundabouts/BasicFacts.htm></sup>
+
+## Roundabout types
+
+### Mini-roundabout
+
+**Mini-Roundabout** are smaller than roundabouts with _right-of-way_ and a _[traversable](https://wiki.openstreetmap.org/wiki/Traversable)_ centre island. These are also often replaced by stop signs at four-way intersections typically used in residential neighborhoods to slow traffic speeds and reduce accidents and not designed to accommodate larger vehicles. Vehicles are required to treat the painted circle as if it were a solid island and drive around it.
+
+![Mini-roundabout](https://c2.staticflickr.com/6/5543/9459511782_dbfacf91f3_b.jpg)
+
+<sup>_A view of a mini roundabout from Church Lane with School Lane to the right and Church Street (A 623) to the left with a partial view of Saint Annes Church (Church of England) and War Memorial in the village of Baslow in Derbyshire. Photo Taken: 10/07/13_</sup>
+
+<sup>_by [Eamon Curry](https://www.flickr.com/photos/eamoncurry/) - <https://flic.kr/p/fpUthJ>_</sup>
+
+**Mini-roundabouts** can be a painted circle or a low dome but **must be fully traversable** by vehicles. Mini-roundabouts are sometimes grouped in pairs (a double mini-roundabout) or in "chains", simplifying navigation of otherwise awkward junctions. Vehicles are required to treat the painted circle as if it were a solid island and drive around it.
+
+![Mini-roundabout chaine](http://s0.geograph.org.uk/geophotos/03/06/29/3062935_47b91623.jpg)
```

Comment:
This image shows a give-way sign. This is in contrast to the explanation above where we call out the right-of-way. If you want to keep this image I would specifically call out exceptions like this give-way sign here.

Reply:
The main purpose of this image is to show `chained` mini-roundabouts when one comes next to another.

`give way` sign is applied to approaching traffic setting a higher priority to the traffic on the mini-roundabout.